recipes/libcap/all/conanfile.py

```diff
@@ -45,6 +47,7 @@ def _configure_autotools(self):
         if self._autotools:
             return self._autotools, self._autotools_env
         self._autotools = AutoToolsBuildEnvironment(self)
+        self._autotools.fpic = self.options.get_safe("fPIC", True)
```

Comment:
I think it's automanaged, but let's keep as is

Reply:
Only if fPIC option still exists. If I remove this line, it fails if shared: https://github.com/conan-io/conan-center-index/pull/5047#issuecomment-807343918